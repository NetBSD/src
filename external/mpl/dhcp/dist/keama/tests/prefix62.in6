# DHCPv6 prefix config

# subnet declaration
subnet6 2001::/64 {
    # range declaration
    option dhcp6.domain-search "example.com", "example.org";
    default-lease-time 1800;
    prefix6 2001:0:0:1:: 2001:0:0:3:: / 64;
    interface "en0";
}
