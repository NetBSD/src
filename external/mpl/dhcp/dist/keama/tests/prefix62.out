{
  # DHCPv6 prefix config
  # subnet declaration
  "Dhcp6": {
    "subnet6": [
      {
        "id": 1,
        "subnet": "2001::/64",
        "option-data": [
          # range declaration
          {
            "space": "dhcp6",
            "name": "domain-search",
            "code": 24,
//          "original-data": "\"example.com\", \"example.org\"",
            "data": "example.com, example.org"
          }
        ],
        "valid-lifetime": 1800,
        "pd-pools": [
//        {
//          "prefix": "2001:0:0:1::",
//          "delegated-len": 64,
//          "prefix-highest": "2001:0:0:3::"
//        }
        ],
        "interface": "en0"
      }
    ],
    "interfaces-config": {
      "interfaces": [
        "en0"
      ]
    }
  }
}
